modules/hcp-worker-hc.adoc

```diff
@@ -8,109 +8,42 @@
-  -P name=<hosted_cluster_name>-worker0 // <7>
-----
-+
-<1> Include `start=False` if you do not want the virtual machine (VM) to automatically start upon creation.
-<2> Include `uefi_legacy=true` to indicate that you will use UEFI legacy boot to ensure compatibility with previous UEFI implementations.
-<3> Replace `<hosted_cluster_name>` with the name of your hosted cluster. The `plan=<hosted_cluster_name>` statement indicates the plan name, which identifies a group of machines as a cluster.
-<4> Include the `memory=8192` and `numcpus=16` parameters to specify the resources for the VM, including the RAM and CPU.
-<5> Include `disks=[200,200]` to indicate that you are creating two thin-provisioned disks in the VM.
-<6> Include `nets=[{"name": "<network>", "mac": "aa:aa:aa:aa:02:13"}]` to provide network details, including the network name to connect to, the type of network (`ipv4`, `ipv6`, or `dual`), and the MAC address of the primary interface.
-<7> Replace `<hosted_cluster_name>` with the name of your hosted cluster.
+* Install a {op-system} compute machine on bare-metal infrastructure for use in the cluster.
+* Log in as a user with `cluster-admin` privileges.
+* Create a `Secret` CR for the Bare Metal Host.
 
-.. Enter the following command to create the second virtual machine:
-+
-[source,terminal]
-----
-$ kcli create vm \
-  -P start=False \// <1>
-  -P uefi_legacy=true \// <2>
-  -P plan=<hosted_cluster_name> \// <3>
-  -P memory=8192 -P numcpus=16 \// <4> 
-  -P disks=[200,200] \// <5>
-  -P nets=["{\"name\": \"<network>\", \"mac\": \"aa:aa:aa:aa:11:02\"}"] \// <6>
-  -P uuid=aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaa1102 
-  -P name=<hosted_cluster_name>-worker1 // <7>
-----
-+
-<1> Include `start=False` if you do not want the virtual machine (VM) to automatically start upon creation.
-<2> Include `uefi_legacy=true` to indicate that you will use UEFI legacy boot to ensure compatibility with previous UEFI implementations.
-<3> Replace `<hosted_cluster_name>` with the name of your hosted cluster. The `plan=<hosted_cluster_name>` statement indicates the plan name, which identifies a group of machines as a cluster.
-<4> Include the `memory=8192` and `numcpus=16` parameters to specify the resources for the VM, including the RAM and CPU.
-<5> Include `disks=[200,200]` to indicate that you are creating two thin-provisioned disks in the VM.
-<6> Include `nets=[{"name": "<network>", "mac": "aa:aa:aa:aa:02:13"}]` to provide network details, including the network name to connect to, the type of network (`ipv4`, `ipv6`, or `dual`), and the MAC address of the primary interface.
-<7> Replace `<hosted_cluster_name>` with the name of your hosted cluster.
+.Procedure
 
-.. Enter the following command to create the third virtual machine:
+. In the web console, navigate to *Compute* -> *Bare Metal Hosts*.
+. Select *Add Host* -> *New from YAML*.
+. Copy and paste the below YAML, modifying the relevant fields with the details of your host:
```

Comment:
I would mention that this is a sample, and all the documentaiton regarding the BareMetal Operator could be found in the RedHat official doc:
- https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html/post-installation_configuration/post-install-bare-metal-configuration#the-baremetalhost-spec

Reply:
Done